README.md

```diff
@@ -77,6 +77,22 @@ this driver supports the following:
 > **Note**
 > `msedgedriver` support is limited. `appium:autodownloadEnabled` does not work for the driver, thus `appium:executable` is necessary to automate MSEdge browser properly.
 
+## Scripts
+
+### install-chromedriver
+
+This script is used to install the given or latest stable version of chromedriver from
+[Chorme for testing](https://github.com/GoogleChromeLabs/chrome-for-testing)
+in `node_modules/appium-chromedriver/chromedriver` directory with `appium driver run chromium install-chromedriver`.
+
+As same as [appium-chromedriver](https://github.com/appium/appium-chromedriver),
```

Comment:
not sure if it also would make sense to introduce command line arguments for the same

Reply:
Do you mean giving `CHROMEDRIVER_VERSION` and `CHROMELABS_URL` would not be helpful? So, just gives `appium driver run chromium install-chromedriver` to download latest version?

Reply:
it's fine to have them. I mean that maybe duplicating their functionality by adding --chromedriver-version and --chromelabs-url command line params.

Reply:
Gotcha. I thought similar to past appiun-chromedriver's downloading method (via ENV var) would be easy to use for users who already use this. Let me do it as a followup PR to extend the support the arg method as well.